R/CreatePackageReport.R

```diff
@@ -1,17 +1,18 @@
-#' @title Surface the internal and external dependencies of an R package.
+#' @title pkgnet Analysis Report for an R package
 #' @name CreatePackageReport
-#' @description Surface the internal and external dependencies of an R package.
-#' @author B. Burns
-#' @seealso GetPackageGraphs
+#' @description Create a standalone HTML report about a package and its networks.
+#' @param pkg_name (string) name of a package
+#' @param pkg_path (string) The path to the package repository. If given, coverage
+#'                 will be calculated for each function. \code{pkg_path} can be an
+#'                 absolute or relative path.
 #' @param pkg_reporters (list) a list of package reporters
 #' @param report_path (string) The path and filename of the output report.  Default
 #'                   report will be produced in the temporary directory.
-#' @inheritParams doc_shared
 #' @importFrom assertthat assert_that is.readable is.string is.writeable
 #' @importFrom methods is
 #' @importFrom tools file_ext
 #' @importFrom utils browseURL
-#' @importFrom knitr opts_chunk
```

Comment:
@jayqi We should keep `knitr::opts_chunk` referenced here for Roxygen docs NAMESPACE since it is used in `package_report.Rmd`.

Reply:
👍 I just added a NULL right underneath `CreatePackageReport` with the `@importFrom knitr` block. (I changed my mind a little bit from #119. I think keeping it slightly separate from the `CreatePackageReport` imports with additional comments makes it more clear where it came from.)